addons/bestja_detailed_reports/views/detailed_reports.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="bestja_detailed_report">
+            <field name="name">bestja.detailed_report.form</field>
+            <field name="model">bestja.detailed_report</field>
+            <field name="priority" eval="0"/>
+            <field name="arch" type="xml">
+                <form>
+                    <header>
+                        <field name="id" invisible="1"/>
+                        <field name="user_can_moderate" invisible="1"/>
+                        <div attrs="{'invisible': [('id', '=', False)]}">
+                            <button name="set_sent" string="Wyślij raport" states="draft" type="object" class="oe_highlight" attrs="{'invisible': [
+                                '|',
+                                    ('state', 'in', ['sent', 'accepted']),
+                                '|',
+                                    ('id', '=', False),
```

Comment:
`('id', '=', False),` is a repetition of a rule already present in the parent div.